<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translation</class>
 <widget class="QWidget" name="translation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>338</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>刘俊专用翻译</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>输入语言</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>翻译</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="ko">
       <property name="text">
        <string>韩语</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ru">
       <property name="text">
        <string>俄语</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ja">
       <property name="text">
        <string>日语</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="cn">
       <property name="text">
        <string>汉语</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="en">
       <property name="text">
        <string>英语</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>输出结果</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>英语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>俄语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>韩语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>日语</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中文</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit2">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>73</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>171</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../5915124c0d98f57c84c002c1bbf89a8.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cD">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看人物说明</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
